src/status_im/ui2/screens/chat/styles/chat_icon.cljs

```diff
@@ -0,0 +1,167 @@
+(ns status-im.ui2.screens.chat.styles.chat_icon
+  (:require [quo.design-system.colors :as colors]
+            [status-im.ui.components.emoji-thumbnail.utils :as emoji-utils]))
+
+(defn default-chat-icon [color]
+  {:margin           0
+   :width            40
+   :height           40
+   :align-items      :center
+   :justify-content  :center
+   :border-radius    20
+   :background-color color})
+
+(defn default-chat-icon-redesign [color size]
+  {:margin           0
+   :width            size
+   :height           size
+   :align-items      :center
+   :justify-content  :center
+   :border-radius    (/ size 2)
+   :background-color color})
+
+(defn default-chat-icon-chat-list [color]
```

Comment:
I think using `assoc` is recommended because we know the keys and values we want to `merge`, As it's faster .. So we use `assoc`and merge when necessary